### DISCUSSION TITLE
Calling ENV.userpaths! is disabled! There is no replacement.

### DISCUSSION THREAD

**Discussion by lyg2024** [Homebrew/discussions#1292] [Getting started] [answered]
I want install  nginx-full --with-rtmp-module
but it show me 
Error: An exception occurred within a child process:
  MethodDeprecatedError: Calling ENV.userpaths! is disabled! There is no replacement.

![image](https://user-images.githubusercontent.com/18829432/115000100-02f89400-9ed5-11eb-9f42-596bc01b86a9.png)

**Comment by SMillerDev** [comment, marked as answer]
You should file an issue with the tap. We don't control the denji/nginx tap.

**Reply by lyg2024** [reply]
I think it not error at denji/nginx , I just want to know : How to enabled  ENV.userpaths

**Reply by SMillerDev** [reply]
You can't, the feature has been removed from brew. The tap maintainer should update the code to work with the latest brew version.

**Reply by lyg2024** [reply]
oo  ,think you ,i get it https://github.com/denji/homebrew-nginx/issues/383 ,I'll have a try.
